packages/sonataflow-management-console-image/resources/incubator-kie-sonataflow-management-console-image.yaml

```
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+- name: "docker.io/apache/incubator-kie-sonataflow-devmode"
+  from: "registry.access.redhat.com/ubi9/httpd-24:1-336.1725850633"
+  version: "0.0.0"
+  description: "SonataFlow Management Console Image"
+
+  labels:
+    - name: "io.quarkus.platform.version"
+      value: "### SET ME DURING BUILD PROCESS ###"
+    - name: "org.kie.kogito.version"
+      value: "### SET ME DURING BUILD PROCESS ###"
+    - name: "maintainer"
+      value: "Apache KIE <dev@kie.apache.org>"
+    - name: "io.k8s.description"
+      value: "SonataFlow Management Console Image."
+    - name: "io.k8s.display-name"
+      value: "Sonataflow Management Console"
+    - name: "io.openshift.tags"
+      value: "sonataflow,serverless,workflow"
+    - name: "io.openshift.expose-services"
+      value: "$KOGITO_MANAGEMENT_CONSOLE_PORT:http"
+
+  modules:
+    repositories:
+      - path: modules
+    install:
+      - name: org.kie.kogito.system.user
+      - name: org.kie.kogito.logging
+      - name: org.kie.kogito.project.versions
+      - name: org.kie.sonataflow.management.console
+
+  ports:
+    - value: $KOGITO_MANAGEMENT_CONSOLE_PORT
```

it doesn't seems to me that it really takes the env var here. If I hard code `8080` I don't have the message:
`["Value 'SONATAFLOW_MANAGEMENT_CONSOLE_PORT' is not of type 'int'. Path: '/value'"]`
 
```suggestion
    - value: $SONATAFLOW_MANAGEMENT_CONSOLE_PORT
```